metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -400,7 +401,11 @@ def _build_conversion_metric_output_node(
             descendent_filter_specs=metric_spec.filter_specs,
             queried_linkable_specs=queried_linkable_specs,
         )
-        entity_spec = EntitySpec.from_name(conversion_type_params.entity)
+        assert StructuredLinkableSpecName.from_name(conversion_type_params.entity).is_element_name, (
```

Comment:
Do we need to raise an error here? Similar to the last one, I don't want any unexpected blockers to show up when this released to MFS. Might be better if we could log it instead.